packages/docusaurus-plugin-content-blog/components/BlogPost/styles.module.css

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+.blogPostContainer {
+  border-bottom: 1px solid #e0e0e0;
+    border-radius: 3px;
+    padding-bottom: 20px;
+    margin: 50px auto;
+    text-align: left;
+    max-width: 1100px;
+}
```

Comment:
Add new line  at end of files. (probably configure your editor to do it).
This file is also mis-indented.

Reply:
Oh my bad. I usually rely on prettier. Just realize we had no prettier setup on `.css`